monkey/common/types/utils/socket_address.py

```diff
@@ -0,0 +1,18 @@
+from ipaddress import IPv4Address
+
+from common.network.network_utils import address_to_ip_port
+from common.types import SocketAddress
+
+
+def socketaddress_from_string(address_str: str) -> SocketAddress:
```

Comment:
Any reason why this is not in the pydantic object? We could then do `SocketAddress.from_string(string)`?

Reply:
This is addressed in [2350](https://github.com/guardicore/monkey/pull/2350)